js/screens/Favourite/Favourite.js

```diff
@@ -11,16 +11,21 @@ const renderSeparator = () => {
 const Favourite = ({ faves }) => {
   return (
     <View style={styles.container}>
-      <FlatList
-        keyExtractor={item => item.listingId}
-        data={faves}
-        ItemSeparatorComponent={renderSeparator}
-        renderItem={({ item }) => (
-          <View>
-            <FavouriteItem item={item} />
-          </View>
-        )}
-      />
+      <Text style={styles.faveText}>
+        {faves.length === 0 ? "You haven't added any favourites yet!" : null}
```

Comment:
You can use if statement here instead of using ternary statement and passing null